docs/source/developer_guide/plugins/basics.rst

```diff
@@ -4,72 +4,28 @@ Basics
 ======
 
 
-What a plugin Is
-----------------
+Nomenclature
+------------
 
-An AiiDA plugin is a `python package <packages>`_ that provides a set of extensions to AiiDA.
+An AiiDA plugin is an extension of AiiDA, usually announcing itself to ``aiida-core`` by means of a new :ref:`entry point <plugins.entry_points>`.
```

Comment:
> usually announcing

Is there another way?

Reply:
I guess, to keep things simple we can define the plugin in this way and remove "usually".

Of course, there are gray areas - I would call [aiida-graphql](https://github.com/dev-zero/aiida-graphql/blob/develop/pyproject.toml) an extension of AiiDA although it does not specify any new entry points.
Perhaps it is not a "plugin" (but of course it is listed on the plugin registry...).

I guess at some point we just have to accept some ambiguity in the terms.

Reply:
I see, fair enough, fine to keep it like this then for now